res/Sectors/M1105/Lorspane.txt

```diff
@@ -30,375 +30,374 @@
+1303                      CA9A422-A Ni Oc                        {  0 } (C36+2) [4467] - -  - 903  9 Hv M2 V M3 V      
+1304                      B898204-8 Lo                           {  0 } (611+1) [3237] - KM - 702  7 Hv M0 V      
+1305                      C442661-6 He Ni Po Cy O:1404           { -2 } (752+2) [B445] - -  - 112  9 Hv M2 V M3 V       
+1307                      C5757S3-5 Ag Pi                        {  0 } (865-1) [8798] - -  - 530 10 Hv A9 V G8 V      
+1308                      C100564-B Ni Va O:1011                 {  0 } (943+0) [951D] - -  - 820  9 Hv M9 V      
+1309                      C676621-3 Ag Ni                        { -1 } (A53+0) [5584] - -  - 103  9 Hv M5 V      
+1402                      B89A7S6-C Wa Pi                        {  2 } (A6D+2) [893B] - -  - 503 10 Hv M0 III M5 V       
+1404 Gemarina             C58A9T6-C Hi Wa Pr                     {  2 } (A8A-2) [BB5F] - -  - 915 15 Hv M3 V M5 V      
+1503                      B338525-8 Ni                           { -1 } (F40-2) [6493] - K  - 404 13 Hv G6 V      
+1601                      B231661-A Na Ni Po Cy O:1402           {  1 } (A56+1) [B71D] - -  - 815 14 Hv G4 IV      
+1603                      B475204-B Lo                           {  1 } (611+2) [1348] - -  - 302 10 Hv M8 V      
+1609                      A410522-D Ni                           {  1 } (D43-1) [666C] - -  - 105 11 Hv A6 V G5 V      
+1702                      C664404-6 Ni Pa                        { -2 } (633-4) [5236] - -  - 822 16 Hv G3 III      
+1708                      E532521-7 Ni Po                        { -3 } (940+2) [2245] - -  - 402  8 Hv M7 V M0 V      
+1801 Saxai                X779312-0 Lo (Saxai) Fo                { -3 } (921-1) [4132] - -  R 624 17 Hv M5 III      
+1809                      E69A505-8 Ni Wa                        { -3 } (943+4) [529C] - -  - 203 15 Hv K5 V M2 V      
+1903                      C898406-7 Ni Pa                        { -2 } (533+4) [3243] - -  - 923 16 Hv G2 IV      
+2001 Aoo                  X9E2103-3 Lo (Aoo) Fo                  { -3 } (301+2) [2134] - -  R 203 12 Hv M3 II M1 V      
+2002                      C300312-B Lo Va                        {  0 } (521-1) [332E] - M  - 703  9 Hv M8 V      
+2201                      C453321-8 Lo Po                        { -2 } (C21+3) [6147] - -  - 605 10 Hv M6 V M6 V      
+2308                      E100304-A Lo Va                        { -1 } (B21+3) [124B] - -  - 505 11 Hv M0 V      
+2502                      A666564-B Ag Ni Ga Pr O:2903           {  2 } (747-2) [1798] - K  - 200 10 Hv M5 V M5 V      
+2504                      A87A322-F Lo Wa                        {  1 } (A21-1) [745E] - -  - 610  8 Hv M9 V      
+2601                      A768321-8 Lo                           { -1 } (921-5) [2298] - -  - 701  8 Hv K5 V      
+2803 Ghavin               A100314-G Lo Va                        {  3 } (F21-1) [16AF] - KM - 624 19 Hv M6 V      
```

Comment:
Per the T5 Second Survey rules this TL should be reduced to F or E

Reply:
Updated